craft_application/util/logging.py

```diff
@@ -0,0 +1,25 @@
+# This file is part of craft_application.
+#
+# Copyright 2023 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License version 3, as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+# SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Logging helpers."""
+import logging
+from typing import Iterable
+
+
+def setup_loggers(names: Iterable[str]) -> None:
```

Comment:
how about `setup_loggers(*names: str)` instead? So we can do `setup_loggers("my_lib")` instead of `setup_loggers(("my_lib",))`